### PR TITLE
Add permissionless context, mainnet2 zkevm validator

### PR DESCRIPTION
**Pull request by asaj**

### CODE REVIEW

File: rust/config/mainnet_config.json [hunk 1]
```diff
@@ -128,6 +128,20 @@
       "index": {
         "from": 25900000
       }
+    },
+    "zkevm": {
```
Comment (nambrot) [inline rust/config/mainnet_config.json:132]: Does this set a weird precedent? We don't really want folks to be adding their config here right? I was imagining that we would track this config somewhere in `infra` to signify that

Comment (asaj) [reply rust/config/mainnet_config.json:132]: I don't think we have a good answer, and this was easiest

File: typescript/sdk/src/consts/chainMetadata.ts [hunk 2]
```diff
@@ -689,6 +689,23 @@ export const zbctestnet: ChainMetadata = {
   isTestnet: true,
 };
 
+export const zkevm: ChainMetadata = {
```
Comment (jmrossy) [inline typescript/sdk/src/consts/chainMetadata.ts:692]: I agree with Nam's point from his review, I'm not certain that including a PI chain here as a 'core' chain is the right thing. For example, it's inclusion here will fool the Explorer/Warp UI into thinking it's not a PI message, and expect it to have scraped db data. Will that be the case? 

If you're aware of the downside and still prefer to include it, I'm okay with it

File: rust/config/mainnet_config.json (hunk 1 above)
Comment (nambrot) [reply rust/config/mainnet_config.json:132]: Why can't we just track this config in `infra` and then insert it via helm?

**Pull request closed without merging**